mathics/builtin/numbers/numbertheory.py

```diff
@@ -109,15 +109,15 @@ def eval(self, n: Integer, evaluation: Evaluation):
 
 
 class DivisorSigma(SympyFunction):
-    """
+    r"""
     <url>
     :Divisor function: https://en.wikipedia.org/wiki/Divisor_function</url> (<url>
     :SymPy: https://docs.sympy.org/latest/modules/functions/combinatorial.html#sympy.functions.combinatorial.numbers.divisor_sigma</url>, <url>
     :WMA: https://reference.wolfram.com/language/ref/DivisorSigma.html</url>)
 
     <dl>
       <dt>'DivisorSigma[$k$, $n$]'
-      <dd>returns σ_$k$($n$)
+      <dd>returns $\sigma_k$($n$)
```

Comment:
To make that the underscore be interpreted as a subindex, all must be enclosed by $...$. Also it is better to use the LaTeX command for $\sigma$.

Reply:
This works for LaTeX but breaks Django.

Reply:
This is how Django docstring looks in master:
![image](https://github.com/user-attachments/assets/8f6a6765-f061-4b23-a5f0-9311ee2158b1)


This is not perfect either. I think we should handle this in the Mathics-Django side.

Reply:
ok.

Reply:
Mathics-Django uses Mathjax to render equations, so it is just a matter to call the render when the documentation is loaded. I will look how to do this later.

Reply:
Ok - good. In the end I guess there will be improvement there as well.

Reply:
Regarding this.

According the MathML documentation, and what we do in
https://github.com/Mathics3/mathics-django/blob/143195d67d3afd2e4cde94f1dde4a641a5fbdd0c/mathics_django/web/media/js/mathics.js#L310


it should be enough to add just bellow here:
https://github.com/Mathics3/mathics-django/blob/143195d67d3afd2e4cde94f1dde4a641a5fbdd0c/mathics_django/web/media/js/doc.js#L6
the line
```
  MathJax.Hub.Queue(['Typeset', MathJax.Hub, doc]);
```

However, for some reason it does not do the trick. Any idea about why, or how to ask?

Reply:
A guess is that this routine is hooked in from a path that processes evaluation output from Python which is distinct from the path that displays document output

Reply:
Yes, it seems something like that happens, but I still could not figure out what/how.